r/tools/linuxlibs.R

```diff
@@ -99,6 +113,23 @@ identify_os <- function(os = Sys.getenv("LIBARROW_BINARY_DISTRO")) {
     cat("*** Unable to identify current OS/version\n")
     os <- NULL
   }
+
+  # Now look to see if we can map this os-version to one we have binaries for
+  os <- find_available_binary(os)
+  os
+}
+
+find_available_binary <- function(os) {
```

Comment:
This lookup was tested successfully here: https://github.com/ursa-labs/arrow-r-nightly/runs/389903019